toolchain/find.go

```diff
@@ -133,11 +134,16 @@ func newInfo(toolchainPath, dir, configFile string) (*Info, error) {
 	}
 
 	prog := filepath.Join(".bin", filepath.Base(toolchainPath))
+
+	if runtime.GOOS == "windows" {
+		prog += ".exe"
+	}
+
 	if fi, err := os.Stat(filepath.Join(dir, prog)); os.IsNotExist(err) {
 		prog = ""
 	} else if err != nil {
 		return nil, err
-	} else if !(fi.Mode().Perm()&0111 > 0) {
+	} else if runtime.GOOS != "windows" && !(fi.Mode().Perm()&0111 > 0) {
```

Comment:
Since Windows doesn't have the same model of permissions as *nix this freaks out. We can generally assume on Windows that a .exe is going to be executable though.